app/models/organizer_position.rb

```diff
@@ -35,6 +38,7 @@ class OrganizerPosition < ApplicationRecord
 
   belongs_to :user
   belongs_to :event
+  belongs_to :contract, optional: true
```

Comment:
maybe clarify the type of contract? because we might want to put termination on this